README.md

```diff
@@ -79,7 +79,7 @@ Thomas' [super power](https://drodio.com/what-is-your-leadership-brand/) is *inv
 Thomas is a productivity buff, generally he wakes early and spends a considerable part of each morning getting organized. This is not to say that organization comes easily to Thomas. He has to work on it every day, but has a very good workflow he would be happy to share with anyone. 
 
 ## Life Long Learner
-Thomas strives to be a lifelong learner. Although he feels like he is slowing down.
+Thomas strives to be a lifelong learner. Although he sometimes feels like he is slowing down, this could be a function of the passage of time and increase in his body of knowledge over the years.
```

Comment:
I thought this sentence felt slightly incomplete and added some assumptions about reasons for feeling like we slow down as we get older.